docs/providers/aws/events/apigateway.md

```diff
@@ -313,6 +314,24 @@ functions:
             identityValidationExpression: someRegex
 ```
 
+You can also use the Request Type Authorizer by setting the `type` property. In this case, your `identitySource` could contain multiple entries for you policy cache.
```

Comment:
Maybe we could add a note saying that `token` is default type? It's not obvious without digging into the code.

Reply:
sure